packages/gatsby-source-wordpress/src/gatsby-node.js

```diff
@@ -88,111 +91,190 @@ exports.sourceNodes = async (
+      normalizer: ({ entities }) => normalize.mapPolylangTranslations(entities),
+    },
+    {
+      name: `createUrlPathsFromLinks`,
+      normalizer: ({ entities }) => normalize.createUrlPathsFromLinks(entities),
+    },
+  ]
 
-  // Creates links between authors and user entities
-  entities = normalize.mapAuthorsToUsers(entities)
-
-  // Creates links between posts and tags/categories.
-  entities = normalize.mapPostsToTagsCategories(entities)
-
-  // Creates links between tags/categories and taxonomies.
-  entities = normalize.mapTagsCategoriesToTaxonomies(entities)
-
-  // Normalize menu items
-  entities = normalize.normalizeMenuItems(entities)
-
-  // Creates links from entities to media nodes
-  entities = normalize.mapEntitiesToMedia(entities)
-
-  // Downloads media files and removes "sizes" data as useless in Gatsby context.
-  entities = await normalize.downloadMediaFiles({
-    entities,
+  const normalizerHelpers = {
     store,
     cache,
     createNode,
     createNodeId,
+    createContentDigest,
     touchNode,
     getNode,
-    _auth,
-    reporter,
-    keepMediaSizes,
-  })
-
-  // Creates links between elements and parent element.
-  entities = normalize.mapElementsToParent(entities)
-
-  // Search and replace Content Urls
-  entities = normalize.searchReplaceContentUrls({
-    entities,
+    typePrefix,
+    refactoredEntityTypes,
+    baseUrl,
+    protocol,
+    _siteURL,
+    hostingWPCOM,
+    useACF,
+    acfOptionPageIds,
+    auth,
+    verboseOutput,
+    perPage,
     searchAndReplaceContentUrls,
-  })
-
-  entities = normalize.mapPolylangTranslations(entities)
-
-  entities = normalize.createUrlPathsFromLinks(entities)
+    concurrentRequests,
+    excludedRoutes,
+    keepMediaSizes,
+  }
 
   // apply custom normalizer
   if (typeof _normalizer === `function`) {
-    entities = _normalizer({
-      entities,
-      store,
-      cache,
-      createNode,
-      createNodeId,
-      touchNode,
-      getNode,
-      typePrefix,
-      refactoredEntityTypes,
-      baseUrl,
-      protocol,
-      _siteURL,
-      hostingWPCOM,
-      useACF,
-      acfOptionPageIds,
-      auth,
-      verboseOutput,
-      perPage,
-      searchAndReplaceContentUrls,
-      concurrentRequests,
-      excludedRoutes,
-      keepMediaSizes,
+    wordpressDataNormalizers.push({
+      name: `customNormalizer`,
+      normalizer: helpers => _normalizer(helpers),
     })
   }
 
   // creates nodes for each entry
-  normalize.createNodesFromEntities({
-    entities,
-    createNode,
-    createContentDigest,
+  wordpressDataNormalizers.push({
+    name: `createNodesFromEntities`,
+    normalizer: ({ entities, createNode, createContentDigest }) =>
+      normalize.createNodesFromEntities({
+        entities,
+        createNode,
+        createContentDigest,
+      }),
   })
 
+  if (typeof _normalizers === `function`) {
+    wordpressDataNormalizers = _normalizers(
+      [...wordpressDataNormalizers],
+      normalizerHelpers
```

Comment:
Should this second parameter be here? If so we should have it documented as well (docs changes just mention first one - `normalizers`